spec/controllers/account_reset/delete_account_controller_spec.rb

```diff
@@ -100,6 +104,136 @@
         t('errors.account_reset.granted_token_expired', app_name: APP_NAME),
       )
     end
+
+    it 'logs info about user verified account' do
+      user = create(
+        :user,
+        :fully_registered,
+        confirmed_at: Time.zone.now.round,
+        profiles: [build(:profile, :active, :verified, pii: { first_name: 'Jane' })],
+      )
+
+      create_list(:webauthn_configuration, 2, user: user)
+      create_account_reset_request_for(user)
+      grant_request(user)
+      session[:granted_token] = AccountResetRequest.first.granted_token
+
+      delete :delete
+
+      expect(@analytics).to have_logged_event(
+        'Account Reset: delete',
+        user_id: user.uuid,
+        success: true,
+        errors: {},
+        mfa_method_counts: {
+          phone: 1,
+          webauthn: 2,
+        },
+        identity_verified: true,
+        account_age_in_days: 0,
+        account_confirmed_at: user.confirmed_at,
+      )
+      expect(response).to redirect_to account_reset_confirm_delete_account_url
```

Comment:
I think we could simplify this a bit:

- The spec is about the logging result for verified accounts, so we can skip repeating assertions about redirects, or behavior when the user has WebAuthn configured
- Rather than manually craft the profile, we could use existing [user factory](https://github.com/18F/identity-idp/blob/main/spec/factories/users.rb) traits (e.g. `:proofed`, `:proofed_with_selfie`, `:proofed_with_gpo`)

Same notes apply to the subsequent tests.

```suggestion
      user = create(:user, :proofed)
      create_account_reset_request_for(user)
      grant_request(user)
      session[:granted_token] = AccountResetRequest.first.granted_token

      delete :delete

      expect(@analytics).to have_logged_event(
        'Account Reset: delete',
        user_id: user.uuid,
        success: true,
        errors: {},
        mfa_method_counts: { phone: 1 },
        identity_verified: true,
        account_age_in_days: 0,
        account_confirmed_at: user.confirmed_at,
      )
```

Reply:
Thanks, that's much easier!